README.md

```diff
@@ -145,9 +145,47 @@ You can then run using that configuration file like so:
 $ docker-app render -f prod.yml
 ```
 
-
 More examples are available in the [examples](examples) directory.
 
+## CNAB
+
+Under the hood `docker-app` is CNAB compliant. It generates a CNAB from your application source and is able to install and manage any other CNAB too.
+CNAB specifies four actions which `docker-app` provides as commands:
+- `install`
+- `upgrade`
+- `status`
```

Comment:
status seem to have been removed from the set of mandatory commands. We need to add this to the list of custom CNAB actions in the created bundles (see https://github.com/deislabs/cnab-spec/blob/master/103-bundle-runtime.md#executing-the-run-tool-cnab-actions)

Reply:
@simonferquel Let's land this and fix in a follow up

Reply:
Aggreed as soon as we keep track of this in an issue

Reply:
Fixed in #451